### PR TITLE
Increases price of CSaber from 18 to 35 materials.

### PR DESCRIPTION
**Pull request by comradef191**
[BALANCE]
**PLEASE DO NOT SHANK ME. I WAS BRIBED BY `Mystic Midgit#6020` TO DO THIS, I HAVE NO OPINIONS ON THIS MATTER.**

## About the PR 
Increases the materials price of the Cyalume caber to 35, as opposed to 18.
For comparison, the crusher is set to 20, and industrial mining armour is set to 45.

## Why's this needed? 
"To hopefully make it harder to mass produce c sabers and hold off the greytide that normally follows after said c sabers are mass produced"

## Changelog

```changelog
(u)Mystic_Midgit
(*)The Cyalume Saber now costs 35 materials, as opposed to its usual 18.
```

### CODE REVIEW

**Comment by unfunnyperson** [comment]
good idea but make it require a resource like plasmastone/erebite because its still really easy to get your hands on steel, glass, and copper

**Comment by ZeWaka** [comment]
yo @Iamgoofball can you review this

File: code/obj/item/misc_weapons.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 	flags = FPRINT | TABLEPASS | NOSHIELD | USEDELAY
 	tool_flags = TOOL_CUTTING
 	is_syndicate = 1
```
Comment (Iamgoofball) [inline code/obj/item/misc_weapons.dm:40]:
```suggestion
	is_syndicate = TRUE
```

**Review by Iamgoofball — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
@ZeWaka done

File: code/obj/item/misc_weapons.dm [hunk 2]
```diff
@@ -38,7 +38,7 @@
...
-	mats = 18
+	mats = 35
```
Comment (Tarmunora) [inline code/obj/item/misc_weapons.dm:41]: Changing this to also some kind of specific/custom mat (even just bohrum or something) would be ideal, I think

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/obj/item/misc_weapons.dm (hunk 2 above)
Comment (pali6) [reply code/obj/item/misc_weapons.dm:41]: ye

**Pull request closed without merging**